flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/MesosApplicationMasterRunner.java

```diff
@@ -601,6 +601,54 @@ else if (recoveryMode == HighAvailabilityMode.ZOOKEEPER) {
 
 		info.setCommand(cmd);
 
+		// Set base container for task manager if specified in configs.
+		String taskManagerContainerName = flinkConfig.getString(
```

Comment:
The variable name (and associated comment) is inaccurate; may I suggest something like `tmImageName`.